### DISCUSSION TITLE
Random Tuxemon Art/Illustrations

### DISCUSSION THREAD

**Discussion by ultidonki** [Tuxemon/Tuxemon#2306] [Show and tell]
Feel free to post any art/illustrations about Tuxemon!!! Anything OK!!!

Here's one I made of Billie and Tuxeman playing Tuxemon!! 
(Who do you think is winning??)

![gaming_tuxemon1](https://github.com/Tuxemon/Tuxemon/assets/89098769/e920e9ae-4ab3-4f73-8c34-4621edc12824)

New illustration by ultidonki, CC-BY-SA-4.0+, based on:
Tuxeman license is: CC-BY-SA-4.0+ - Original design by Leo
Billie license is: CC-BY-4.0 by Catch Challenger - art director Jordan Brule. Project developer Herman Brule. Re-drawn by ultidonki.
Drawn by ultidonki, based on Female trainer which is: CC-BY-4.0 by Catch Challenger - art director Jordan Brule. Project developer Herman Brule.

**Comment by ultidonki** [comment]
I don't have much time to contribute to Tuxemon recently, but I drew something else: :smile:
Billie in the rain, holding an umbrella:

![billie_umbrella](https://github.com/Tuxemon/Tuxemon/assets/89098769/db20ce5b-05ad-4aa2-bc1c-602e50132184)

Billie license is: CC-BY-4.0 by Catch Challenger - art director Jordan Brule. Project developer Herman Brule. Re-drawn by ultidonki.
Drawn by ultidonki, based on Female trainer which is: CC-BY-4.0 by Catch Challenger - art director Jordan Brule. Project developer Herman Brule.

**Comment by HippasusTwo** [comment]
Nice.

I wonder what kind of person would name their child Tuxeman though...

**Comment by Davidliang11292** [comment]
I asked the AI for advice. How about we name the main character Libero? The name comes from the Esperanto root Liber-, which is a good expression for the spirit of freedom and openness

**Reply by keshav-k3** [reply]
Libero sounds great according to me, **Libero Skyridge** or something, metaphor the sky for openness again

**Reply by Davidliang11292** [reply]
Thanks!

**Comment by JaskRendix** [comment]
@ultidonki I tried to generate an opening for your videos/animations. What do you think about this intro?

**Reply by ultidonki** [reply]
Oh, that's actually quite good - you've really captured a good mood with the song there. 

However - it's just a personal choice - I'm trying not to use AI at the minute, so I don't think I'll use it myself. But if other people want to use it, that's fine. I've seen other people use AI for their voices for their animations too, and it's probably better than silence, but I just don't want to yet. Are the lyrics generated using AI too?

Thanks for your support though :smile:

**Comment by ultidonki** [comment]
I've also started working on my next Tuxemon comic, so I'll post that here when it's finished! For the first time, Rockitten doesn't appear in this episode :smiley:

**Reply by Davidliang11292** [reply]
Wow! Looking forward to your new work!
